src/overlays/actors/ovl_En_Dodongo/z_en_dodongo.h

```diff
@@ -5,13 +5,35 @@
 
 struct EnDodongo;
 
-typedef void (*EnDodongoActionFunc)(struct EnDodongo* this, GlobalContext* globalCtx);
+typedef void (*EnDodongoActionFunc)(struct EnDodongo*, GlobalContext*);
 
 typedef struct EnDodongo {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x1B8];
-    /* 0x02FC */ EnDodongoActionFunc actionFunc;
-    /* 0x0300 */ char unk_300[0x514];
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ Vec3s jointTable[31];
+    /* 0x242 */ Vec3s morphTable[31];
+    /* 0x2FC */ EnDodongoActionFunc actionFunc;
+    /* 0x300 */ u8 unk_300;
+    /* 0x302 */ s16 unk_302;
```

Comment:
This seems to be a timer based on usage.

Reply:
What does that mean, I just call it `timer`?

Reply:
Yes, exactly.